src/signals/incident/containers/ExternalReplyContainer/__test__/ExternalReplyContainer.test.tsx

```diff
@@ -0,0 +1,364 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Vereniging van Nederlandse Gemeenten
+import { render, waitFor, screen, act } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { mocked } from 'jest-mock'
+import * as reactRouterDom from 'react-router-dom'
+
+import * as actions from 'containers/App/actions'
+import { withAppContext } from 'test/utils'
+
+import ExternalReplyContainer from '..'
+import * as API from '../../../../../../internals/testing/api'
```

Comment:
might be cleaner to add an alias in web pack's resolve config for internals. not really a blocker